### PR TITLE
Modify Readme with Frontity links

### LINKED ISSUE #16: Add Readme

### PR DESCRIPTION
**Pull request by luisherranz**
Fixes #16

### CODE REVIEW

**Comment by luisherranz** [comment]
@orballo @DAreRodz any suggestions for the Readme? Anything missing?

**Review by orballo — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,128 +1,23 @@
...
 
-- Make excerpt optional
+Clone the main [Frontity](https://github.com/frontity/frontity) repository first. Then, inside the `packages` folder, clone this repository.
 
-#### 1.3.1
+Finally, use `npm run start:pwa` to start the development environment.
 
-- Add url in site info for blogs without home on /
-- Fix problem with missing info on media
+## Changelog
```
Comment (orballo) [inline README.md]: This line is not rendering correctly.

**Review by orballo — COMMENTED** [review]
Small bug in README file

File: README.md (hunk 1 above)
Comment (luisherranz) [reply README.md]: It's probably an irregular space.

Comment (luisherranz) [reply README.md]: Solved in https://github.com/frontity/wp-org-connection/pull/17/commits/54ad55159425c6b9fc8df00cd553db5d4f5f3506

**Review by orballo — APPROVED** [review]

**Comment by luisherranz** [comment]
Merging without @DAreRodz approval.

**Pull request merged**